packages/ui/src/components/vuestic-sass/resources/_variables.scss

```diff
@@ -339,3 +339,15 @@ $display-margins: (
   5: 2.25rem,
   6: 2rem
 ) !default;
+
+// Modals
+$-basic-duration: 0.3s;
```

Comment:
looks weird

Reply:
Yes, but it's pretty much convenient when you want to change the transition duration without breaking anything :)